swingbits/src/main/java/org/oxbow/swingbits/table/filter/TableFilterColumnPopup.java

```diff
@@ -170,14 +162,15 @@ protected JComponent buildContent() {
             JToolBar toolbar = new JToolBar();
             toolbar.setFloatable(false);
             toolbar.setOpaque(false);
-            toolbar.add( new PopupWindow.CommandAction(
-                    bundle.getString( "Clear_ALL_COLUMN_FILTERS" ),
-                    new ImageIcon(getClass().getResource("funnel_delete.png"))) {
-                @Override
-                protected boolean perform() {
-                    return clearAllFilters();
-                }
-            });
+            //removed clear filter for JTable

```

Comment:
In my opinion, we should not just remove it, but have API to enable/disable it with default being disabled